### ISSUE TITLE
ts 1.8.10

### ISSUE THREAD

**Issue by thasner** [palantir/eclipse-typescript#319]
Can you bump the bundled Typescript version from 1.8.9 to 1.8.10?

We'd like to avoid the following React error:
```
Warning: React.__spread is deprecated and should not be used. Use Object.assign directly or another helper function with similar semantics. You may be seeing this warning due to your compiler. See https://fb.me/react-spread-deprecation for more details.
```
